conwayconsole/Program.cs

```diff
@@ -27,8 +27,9 @@ public static async Task Main(params string[] args)
           int width = options.Width ?? Console.WindowWidth - 1;
           int height = options.Height ?? Console.WindowHeight - 2;
           var random = options.Seed.HasValue ? new Random(options.Seed.Value) : new Random();
+          double denisty = 1- Math.Clamp(options.Density,0,1);
```

Comment:
typo "density"

Reply:
Typo is fixed, but I have just noticed that you deleted the `random` argument to `Randomise()` meaning that Joel's work to allow the seed to be specified has been broken. Please reinstate the `random` argument to `Randomise()`